docs/manage-search-index.md

```diff
@@ -0,0 +1,50 @@
+# Manage Search Index
+
+Each document is stored in the index, and is the type and ID. The document is a JSON object that contains zero or more fields or key-value pairs. The original JSON document indexing will be stored in the field_source that returns a default receipt or document search.
+
+The analysis is performed by а custom analyzer defined by index. Token filter:
```

Comment:
What analysis?

Reply:
The same as above! I don't understand this at all- I'll better give you the link. https://www.slideshare.net/SelectoCompany/search-engine-elasticsearch

Reply:
Let's replace this intro with the following:

VirtoCommerce uses custom analyzer to extract tokens from the text when building search index. Following token filters are supported:
- custom_edge_ngram (default) 
- custom_ngram

Reply:
OK, this is done! What about the mapping? Should we keep it there?

Reply:
I think we should remove the sentence about mapping from there.

Reply:
Agree, lets remove.

Reply:
Done!